src/garage/experiment/local_runner.py

```diff
@@ -169,6 +171,12 @@ def setup(self, algo, env, sampler_cls=None, sampler_args=None):
             sampler_cls = algo.sampler_cls
         self._sampler = sampler_cls(algo, env, **sampler_args)
 
+        # pylint: disable=import-outside-toplevel
```

Comment:
this is ugly. i think that the algorithm should do evaluation sampling instead

@krzentner